doc/cortx-aws-k8s-installation.md

```diff
@@ -457,6 +457,56 @@ chmod +x s3-benchmark
 | S3 | Multiple IPs (cortx-server-clusterip-svc pods) | tcp/443, tcp/80
 | IAM | Multiple IPs (cortx-server-clusterip-svc pods) | tcp/9443
 
+## 6 CloudFormation Guide
+
+The steps in this guide can be automated using AWS CloudFormation.
```

Comment:
So, I'm not familiar with this full guide. Did you add a new section starting at line 460 which is a simpler way to do everything that is between lines 0-460? If so, wouldn't it be better to delete the old instructions which do it the complex way and replace the entire contents of this doc with only the newer better way?
Or, are the full instructions still needed? If so, will the majority of readers prefer the old way or the new way? Whichever is the preference should be first. And at the top it should say, "If you prefer the blah blah way, jump to this section(LINK)"

But I suspect we need to just delete the old way and replace it with the new simpler way. No?

Reply:
There's still useful information in the full guide on how to choose/customize the configuration, and explanation about what's happening, so I wouldn't want to simply delete the rest. The new way is certainly easier, so if you think the majority of readers will just use CloudFormation, I can greatly shorten the guide